R/timeseriessummary-data.R

```diff
@@ -413,8 +413,14 @@ unNestCorrectionParameters <- function(corrections, timezone) {
   deviationType <- ".dplyr"
   windowSizeInMinutes <- ".dplyr"
   ungroup <- ".dplyr"
+  
+  if (corrections[["type"]]=="CopyPaste" || corrections[["type"]]=="Freehand" || corrections[["type"]]=="DeleteRegion" || corrections["type"]=="RevertToRaw") {
+    corrections[["parameters.dummy"]] <- ""
```

Comment:
I would do an if-else kind of thing down by cbind instead. Then corrections would remain the same if params was empty.

```r
if(!isEmptyOrBlank(params)){
    corrections <- cbind(corrections, params)
} 
```